docs/src/main/sphinx/release/release-393.md

```diff
@@ -0,0 +1,141 @@
+# Release 393 (10 August 2022)
+
+## General
+
+* Add support for {doc}`/sql/merge`. ({issue}`7933`)
+* Improve performance of highly selective `LIMIT` queries by making them finish
+  as soon as the required number of rows is produced. ({issue}`13616`)
+* Prevent using an expression which evaluates to null to `INSERT` into a
+  non-nullable column. ({issue}`13462`)
+* Reduce verbosity of query processing log by moving timeline messages to the
+  debug log level. ({issue}`13668`)
+* Fix potential query stalling on bucketed tables. ({issue}`13655`)
```

Comment:
By "stalling", do we mean a slowdown or the query locks up?

Reply:
Query locked up.

Reply:
Locks up.

Reply:
Then it needs to be phrased more explicitly. How about:

> Fix queries over bucketed tables never completing.